zowe-cli-id-federation-plugin/src/api/Commands.ts

```diff
@@ -39,38 +39,39 @@ export class Commands {
             throw new ImperativeError({msg: "Error when trying to create the identity mapping."});
         }
         if(this.refreshCommand != null){
+            commands.push("");
```

Comment:
Why?

Reply:
I thought it would be good to visually separate the refresh command from the map commands. If you imagine having tens or a hundred of map commands ... It would be easier for a sysprog to validate the jcl before submitting.

Reply:
It is fine I just wasn't sure if it doesn't break JCL :-)